app/src/main/java/com/google/samples/apps/sunflower/compose/home/HomeScreen.kt

```diff
@@ -61,9 +73,55 @@ enum class SunflowerPage(
     PLANT_LIST(R.string.plant_list_title, R.drawable.ic_plant_list_active)
 }
 
+@Composable
+fun HomeScreen(
+    onPlantClick: (Plant) -> Unit,
+    viewModel: PlantListViewModel = hiltViewModel()
+) {
+    val activity = (LocalContext.current as AppCompatActivity)
+
+    val menuProvider = object : MenuProvider {
```

Comment:
I wonder if all of this + the page changes below could be extracted out of HomeScreen and to App level? In that case, you'd just need to pass callbacks to HomeScreen and HomePagerScreen to make the composable super flexible and customizable for this. Dont think its essential tho

Reply:
That's a good idea - that way it's clearer why `AppCompatActivity` is necessary as a superclass. Updated.